### ISSUE TITLE
just the glow effect

### ISSUE THREAD

**Issue by tbjorkna** [slammayjammay/hyper-postprocessing#44]
Hey,

would it be possible to just get the glow effect as you have in some of the examples there? I would like to have a similar theme as on VS code https://marketplace.visualstudio.com/items?itemName=RobbOwen.synthwave-vscode using this glow effect.

Using node v18.17.0

**Comment by slammayjammay** [comment]
Hey, I don't plan to add any more examples here. That example you provided doesn't use fragment shaders, it just has css for that effect. If you haven't already you can search for an existing Hyper theme that you like https://github.com/bnb/awesome-hyper or if you know css you can copy+paste some of that in to your `.hyper.js` config. Closing since I won't be adding this to the repo.

**Comment by tbjorkna** [comment]
I took this example and just commented out those lines to get rid of the space effect and solely get the glow effect on the text https://github.com/slammayjammay/hyper-postprocessing/blob/master/examples/effects/space-travel/index.js#L27-L31 

I discovered that the terminal becomes buggy if you resize it. Seems to be the same issue with other example effects. Have you encountered any issues when resizing the terminal window?
